camunda-formio-tasklist-vue/src/services/SocketIOServices.ts

```diff
@@ -13,23 +18,55 @@ const isConnected = () => {
   return stompClient?.connected || null;
 };
 
-const connect = (encryptKey: any, reloadCallback: any) => {
+const clientConnectCallback = () => {
+  console.log(
```

Comment:
Shouldn't we remove these console logs?

Reply:
already added in notes,this PR is for testing ` added /* eslint-disable */ for console.log() for debugging, once this is tested, will clean the code and created a new PR`